wsd/RequestVettingStation.cpp

```diff
@@ -15,7 +15,9 @@
 
 #include <COOLWSD.hpp>
 #include <TraceEvent.hpp>
+#if !MOBILEAPP
```

Comment:
I hate more conditionals. Can't we stub this and/or have some no-op backend for the Mobile case? we really have waaay too many conditionals in the code, and it makes testing a nightmare. Do we really need these ?